covid19_sfbayarea/news/san_francisco.py

```diff
@@ -59,4 +59,6 @@ def parse_news_item(self, item: element.Tag, base_url: str) -> NewsItem:
         date_string = item.find('time')['datetime']
         date = dateutil.parser.parse(date_string)
 
-        return NewsItem(id=url, url=url, title=title, date_published=date)
+        summary = item.find('div').find('div').find('div').get_text(strip=True)
```

Comment:
Two quick thoughts here:

1. Would you please use `news.utils.normalize_whitespace(element.get_text())` instead of `element.get_text(strip=True)`? I realized a while back that `get_text(strip=True)` is actually a bad pattern in most cases, but I guess I failed to fix it everywhere (totally understand if you were following the example of `title = ...` above, which is my fault for not updating).

2. Is there any better sort of selector we can build from classes, or maybe take the next sibling after the title heading, rather than just going 3 divs deep? This strikes me as pretty error-prone.